packages/client/src/helpers/fetchApi.js

```diff
@@ -1,6 +1,16 @@
+import store from '@/store';
+
+function getDefaultHeaders() {
+  const headers = new Headers();
+  headers.append('agency-id', store.getters['users/selectedAgencyId']);
```

Comment:
I know that Michael had originally described centralization using request headers, but I meant for the query string to be used instead. I really think the most RESTful way to do this would be something more like prefixing `:agencyId/` to all routes that don't already have an agency param or body field. Since that's likely too much effort, I thought the next most RESTful thing was to keep the agency ID in the URL, through the query string.

If you prefer to stay with the header, then the query string approach isn't really used at all, so it should probably be removed. That would have some testing impact, which I'll describe in another comment.

Reply:
Michael had some concerns about refactoring and how the code/express routes would work prefixing agency id in the params. I think ill take a stab at it since at the top of my mind I dont see a lot of refactoring.

Essentially, all routes will be prefixed by:

organizations/:orgId

Thoughts?

Reply:
That approach is:
1. the best (IMO) if we ignore the effort it costs
2. possibly very costly in terms of effort

If you can implement it with effort you're OK with, I'n all for it. It would require a lot of effort to rework test code, but I would be willing to invest that.